admin-front/src/components/Nfts/Selector/types.ts

```diff
@@ -0,0 +1,16 @@
+export interface InputSelectorProps {
+  attributesName: string;
+  label: string;
+  type:
+    | 'string'
```

Comment:
This can also be moved into a named type, i've seen it being referenced in a couple places and in general it belongs under a dedicated type (can be done in a later PR if you prefer that. More a general note, not really a comment on the work in this PR).

Probably a const assertion is applicable here (not sure if I am using correct terminology, but I am referring to what is mentioned here https://stackoverflow.com/a/62694420)?

Reply:
I agree. I have put it into its own type now but if the type will be used outside of these selector components I will make it more general.